web/styles/zulip.css

```diff
@@ -1229,16 +1229,18 @@ nav {
     text-align: center;
 }
 
-div.topic_edit_spinner {
+div.topic_edit_spinner,
+div.toggle_resolve_topic_spinner {
     display: flex;
     align-items: center;
     width: 1.2em; /* 18px at 15px/em  (from font size set in .message-header-contents) */
     height: 1.2em; /* 18px at 15px/em */
 }
 
+.on_hover_topic_unresolve,
 div.toggle_resolve_topic_spinner {
-    margin-top: -12px;
-    padding-left: 9px;
+    width: 1em;
+    padding: 0 6px; /* matching padding from .recipient_bar_icon */
```

Comment:
Should the element just have the `recipient_bar_icon` class instead?

Reply:
Great point, I'll do that, thanks!